scalding-commons/src/main/scala/com/twitter/scalding/commons/source/LzoGenericScheme.scala

```diff
@@ -102,15 +118,7 @@ class LzoGenericScheme[M](@transient conv: BinaryConverter[M], clazz: Class[M])
     tap: Tap[JobConf, RecordReader[_, _], OutputCollector[_, _]],
     conf: JobConf): Unit = {
 
-    val extern = Externalizer(conv)
-    try {
-      ExternalizerSerializer.inj.invert(ExternalizerSerializer.inj(extern)).get
-    } catch {
-      case e: Exception => throw new RuntimeException("Unable to roundtrip the BinaryConverter in the Externalizer.", e)
-    }
-
-    conf.set(ConfigBinaryConverterProvider.ProviderConfKey, ExternalizerSerializer.inj(extern))
-
+    LzoGenericScheme.setConverter(conv, conf)
```

Comment:
Exactly, if something else has gotten there first and wanted to set how the deserialization between the two fixed types is done we don't want to override. I'd view this as the default path per say. 

Use case so far is that I can swap out different thrift implementations for testing by setting this param